InventorySalesApi/Api/Data/AreProductsSalableResultInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Represents result of "are products salable" interface.
+ *
+ * @api
+ */
+interface AreProductsSalableResultInterface extends ExtensibleDataInterface
```

Comment:
I believe we can reuse `\Magento\InventorySalesApi\Api\Data\ProductsSalableResultInterface` here and remove the new interface

Reply:
The reason I've introduced new interface described bellow.